src/components/WriteEditModal/WriteEditModal.tsx

```diff
@@ -47,12 +48,19 @@ export default function WriteEditModal({
 }: WriteEditModalProps) {
   const queryClient = useQueryClient();
 
-  const containerRef = useRef<HTMLDivElement>(null);
-
   const [deletedImageId, setDeleteImageId] = useState<number[]>([]);
+  const [clickedFeedback, setClickedFeedback] = useState<Array<number>>([0, 0, 0]);
+  const [rating, setRating] = useState<number>(0);
+  const containerRef = useRef<HTMLDivElement>(null);
 
   const isCustom = reviewType === 'customReview' || reviewType === 'customReviewEdit';
 
+  const handleClickFeedback = (optionIndex: number, feedbackIndex: number) => {
+    const updatedClickedFeedback = [...clickedFeedback];
+    updatedClickedFeedback[optionIndex] = feedbackIndex;
+    setClickedFeedback(updatedClickedFeedback);
```

Comment:
이것도 내부 값이 코드 실행 도중에 바뀌는 것도 염두해서 setState내부에서 처리하는 게 좋을 것 같아요! 이런 식으로...?
```suggestion
    setClickedFeedback((prev) => prev.map((feedback, i) => i === optionIndex ? feedback : feedbackIndex)
```